src/soilId/soilIdTypes.ts

```diff
@@ -50,6 +52,12 @@ export const collectionMethods = [
 export type SoilPitMethod = (typeof soilPitMethods)[number];
 export type CollectionMethod = (typeof collectionMethods)[number];
 
+export type MeasurementUnit = SoilIdProjectSoilSettingsMeasurementUnitsChoices;
+export const MEASUREMENT_UNITS = [
+  'METRIC',
+  'IMPERIAL',
```

Comment:
question: we're calling this English in the UI. Can we match here?

Reply:
we could, we should soon since this enum is defined in the backend and would require a data migration, but are you ok with doing it in a follow-on PR since the related mobile client PR which would be affected is already very big?

Reply:
A follow-on PR is fine.

Reply:
https://github.com/techmatters/terraso-backend/issues/1212